src/components/roles/RoleEditDialog.vue

```diff
@@ -3,189 +3,187 @@
-              :error-messages="errors"
-            />
-          </validation-provider>
-          <validation-provider
-            v-slot="{ errors }"
-            rules="requiredSelect"
-            name="local group"
-          >
-            <v-select
-              v-model="role.localGroupId"
-              :items="localGroups"
-              item-value="id"
-              item-text="title"
-              :label="$t('Local Group')"
-              :error-messages="errors"
-            />
-          </validation-provider>
-          <validation-provider
-            v-slot="{ errors }"
-            rules="requiredSelect"
-            name="working circle"
-          >
-            <v-select
-              v-model="role.workingCircleId"
-              :items="workingCircles"
-              item-value="id"
-              item-text="title"
-              :label="$t('Working circle')"
-              :error-messages="errors"
-            />
-          </validation-provider>
-          <p style="color: gray" class="caption">
-            {{ $t("Responsibilities") }}
-          </p>
-          <validation-provider
-            v-slot="{ errors }"
-            :rules="`${role.responsibilities.length < 1 ? 'requiredList' : ''}`"
-            mode="eager"
-            name="responsibility"
-          >
-            <v-text-field
-              solo
-              v-model="newResponsibility"
-              :label="$t('Add new responsibility')"
-              :error-messages="errorResponsibility || errors"
-              @keypress.enter="addResponsibility"
-            >
-              <template v-slot:append>
-                <v-btn
-                  text
-                  icon
-                  small
-                  color="primary"
-                  :disabled="!validResponsibility"
-                  @click="addResponsibility"
-                >
-                  <v-icon>mdi-plus</v-icon>
-                </v-btn>
-              </template>
-            </v-text-field>
-          </validation-provider>
-          <v-card v-if="role.responsibilities.length > 0" class="mb-4">
-            <template v-for="(responsibility, i) in role.responsibilities">
-              <v-divider v-if="i !== 0" :key="`${i}-divider`" />
-              <v-list-item
-                :key="`${i}-${responsibility}`"
-                class="d-flex justify-space-between"
-              >
-                <span style="flex: 1">{{ responsibility }}</span>
-                <v-btn
-                  text
-                  icon
-                  color="gray"
-                  @click="role.responsibilities.splice(i, 1)"
-                >
-                  <v-icon>mdi-close</v-icon>
-                </v-btn>
-              </v-list-item>
-            </template>
-          </v-card>
-          <validation-provider
-            v-slot="{ errors }"
+          <multi-language-input
+            v-model="role.title"
+            rules="required"
```

Comment:
I would add the rules it had before. This prevents people from entering extremely long titles that don't fit in the role overview card. Also it's good to prevent special characters from appearing in the title.
```suggestion
            rules="required|alpha_spaces|max:30"
```